sqlmesh/dbt/model.py

```diff
@@ -294,9 +294,13 @@ def to_sqlmesh(self, context: DbtContext) -> Model:
 
             table_properties = {}
             if self.partition_expiration_days is not None:
-                table_properties["partition_expiration_days"] = self.partition_expiration_days
+                table_properties["partition_expiration_days"] = exp.convert(
+                    self.partition_expiration_days
+                )
             if self.require_partition_filter is not None:
-                table_properties["require_partition_filter"] = self.require_partition_filter
+                table_properties["require_partition_filter"] = exp.convert(
+                    self.require_partition_filter
```

Comment:
what is require_partition_filter? what is expected here?

Reply:
This is a boolean value

Reply:
you shouldn't need to call convert here because it was called in the validator